console/console-init/ui/src/modules/iot-certificates/components/CertificateCard/CertificateCard.tsx

```diff
@@ -111,12 +120,12 @@ export const CertificateCard: React.FunctionComponent<ICertificateCardProps> = (
             </Title>
           </GridItem>
           <GridItem span={10} className={rowMargin}>
-            <Switch
-              id={`cc-auto-provision-switch-${id}`}
+            <SwitchWithToggle
+              id={`cf-auto-provision-switch-${id}`}
               label="Enabled"
               labelOff="Disabled"
-              readOnly
               isChecked={certificate["auto-provisioning-enabled"] || false}
+              onChange={onEnableChange}
```

Comment:
The card should be read only, it makes more sense to make any updates in the CertificateForm.

Reply:
I think we should add action on switch button